src/Model/Wijkanders.hs

```diff
@@ -59,11 +63,19 @@ hasDate d =
 getWijkanders :: Day -> ByteString -> Either NoMenu [Menu]
 getWijkanders d =
   parseTags
-        -- Take tags from a start date to the end of that paragraph.
```

Comment:
Rather than betting that they will always keep their "subect to change" notice, why don't we attempt to always find the section (not HTML section, that would have been to easy) starting with "<strong>Dayname day/month</strong>" and continuing to the next such section? That should be somewhat foolproof as to find what their menus for the day.

Reply:
That's exactly how the algorithm works, but we won't find a next day after fridays, that's why I check for that string. So either we take until tomorrow or until the magical phrase. Example: `BL.isPrefixOf (BL8.pack "Med reservation") s || hasDate tomorrow s`, context at line 73.